### PR TITLE
Minor tweaks to build status page

### PR DESCRIPTION
**Pull request by ErikSchierboom**
- Remove comments
- Build page student count is consistent with header
- Remove unused data

### CODE REVIEW

File: app/views/tracks/build/show.html.haml [hunk 1]
```diff
@@ -8,28 +8,25 @@
...
-        .highlight-text= number_with_delimiter(@status.students.num_students)
+        .highlight-text= number_with_delimiter(@track.num_students)
         students have developed their #{@track.title} skills on Exercism.
       .grid.md:grid-cols-3.sm:grid-cols-2.grid-cols-1.mb-32.gap-16
         .report-stat
           .stat-name
             Students
             = graphical_icon "info-circle"
-          .current-number= number_with_delimiter(@status.students.num_students)
-          -# .growth +#{@status.students.num_students_per_day} per day
```
Comment (ErikSchierboom) [inline app/views/tracks/build/show.html.haml:19]: I'm not a fan of commented out code. We can always re-add this via version control

File: app/views/tracks/build/show.html.haml [hunk 2]
```diff
@@ -8,28 +8,25 @@
...
+          .current-number= number_with_delimiter(@track.num_student)
         .report-stat
           .stat-name
             Submissions
             = graphical_icon "info-circle"
           .current-number= number_with_delimiter(@status.submissions.num_submissions)
-          -# .growth +#{@status.submissions.num_submissions_per_day} per day
```
Comment (ErikSchierboom) [inline app/views/tracks/build/show.html.haml:25]: I'm not a fan of commented out code. We can always re-add this via version control

File: app/views/tracks/build/show.html.haml [hunk 3]
```diff
@@ -8,28 +8,25 @@
...
         .report-stat
           .stat-name
             Mentoring Discussions
             = graphical_icon "info-circle"
           .current-number
             = number_with_delimiter(@status.mentor_discussions.num_discussions)
-          -# .growth +10 per day
```
Comment (ErikSchierboom) [inline app/views/tracks/build/show.html.haml:32]: I'm not a fan of commented out code. We can always re-add this via version control

File: app/views/tracks/build/show.html.haml [hunk 4]
```diff
@@ -8,28 +8,25 @@
   .lg-container.grid.grid-cols-3.gap-48.relative.mt-40
     .col-span-3.xl:col-span-2
       %h1.text-h1.mb-16.text-40.font-bold.z-1
-        .highlight-text= number_with_delimiter(@status.students.num_students)
+        .highlight-text= number_with_delimiter(@track.num_students)
```
Comment (ErikSchierboom) [inline app/views/tracks/build/show.html.haml:11]: This makes the student count consistent with the header: https://forum.exercism.org/t/inconsistent-number-of-students-in-track/6518

File: app/commands/track/update_build_status.rb [hunk 5]
```diff
@@ -62,22 +61,9 @@ def volunteers
...
-    {
-      num_students: track.num_students,
-      num_students_per_day:
-    }
-  end
-
-  def num_students_per_day
-    query = MetricPeriod::Day.where(metric_type: Metrics::JoinTrackMetric.name, track:)
-    average(query.sum(:count), query.count)
-  end
```
Comment (ErikSchierboom) [inline app/commands/track/update_build_status.rb:75]: We're not using this anymore, so why calculate it

File: app/commands/track/update_build_status.rb [hunk 6]
```diff
@@ -62,30 +61,12 @@ def volunteers
...
-      num_submissions:,
-      num_submissions_per_day:
+      num_submissions:
     }
   end
 
-  def num_submissions_per_day
-    query = MetricPeriod::Day.where(metric_type: Metrics::SubmitSubmissionMetric.name, track:)
-    average(query.sum(:count), query.count)
-  end
```
Comment (ErikSchierboom) [inline app/commands/track/update_build_status.rb:87]: We're not using this anymore, so why calculate it

**Review by iHiD — APPROVED** [review]

**Pull request merged**